geomap/package-lock.json

```diff
@@ -0,0 +1,9665 @@
+{
```

Comment:
AFAIK the UI team is using yarn, and this already has a yarn lockfile. You shouldn't be using `npm` to create a `package-lock.json` file.